openedx/features/discounts/applicability.py

```diff
@@ -28,6 +28,18 @@
 from common.djangoapps.track import segment
 
 
+# .. toggle_name: discounts.enable_first_purchase_discount_override
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable the First Purchase Discount to be overriden from
+#   EDXWELCOME/BIENVENIDOAEDX 15% discount to a new code with 30% discount.
+# .. toggle_use_cases: temporary
```

Comment:
I added as `temporary`, and I do realize that the majority of the time we add this in and it really stays forever. The other options I see are `open edx` `opt_in` `opt_out`.

There is a date to remove the coupon from visitibility to learners but not necessarily the waffle flag.

Reply:
`opt_in` may also be a good choice as I'm guessing this will default to off, especially where we will want to use this waffle flag/setting again in the future

Reply:
I thought `opt_in` and `opt_out` had to do with some user action to opt in or out, but maybe not